docs/release-notes/2.41.0.md

```diff
@@ -0,0 +1,75 @@
+
+PMM users can now investigate challenging and complex problems, thus providing a deeper understanding of database and network issues.
+
+### Streamlined database problem reporting to Percona
+
+Effective database problem resolution relies on the quality and comprehensiveness of issue reporting.
+
+To improve the process of gathering and sharing PMM metrics and data, we've now integrated the [pmm_dump client utility](https://docs.percona.com/pmm-dump-documentation/index.html) into PMM. Originally a standalone client for PMM Server, this now readily accessible within the PMM user interface.
+
+This integration enables you to effortlessly collect data from PMM and empowers our Support team to better assist customers in solving database issues in SLA and address community-reported bugs in Jira.
+
+To get started, go to the **Help** section on the main menu > **PMM Dump** and choose to either export a dataset locally or upload it via an SFTP server using the credentials generated through your Percona Support ticket.
+
+### PostgreSQL monitoring performance improvement
+
+In our continuing efforts to improve PMM, we recognize the significance of performance and its impact on our users. Starting with PMM 2.41.0, we have optimized resource usage for PostgreSQL users. This has been accomplished by reducing the number of connections used in monitoring.
+
+### PMM DBaaS functionality evolution into Percona Everest
+
+We have achieved positive outcomes in developing PMM's Database-as-a-Service feature. As we move to the next level, we have decided to separate the DBaaS functionality into an independent product. Consequently, we are discontinuing the DBaaS functionality in PMM and offering a [migration path to Everest](http://per.co.na/pmm-to-everest-guide).
+
+While the DBaaS functionality will remain available in PMM v2, all future updates and enhancements will be exclusively accessible through the Percona Everest interface.
+We strongly recommend you try [Percona Everest](http://per.co.na/pmm-to-everest) for a more streamlined and robust experience with database deployments.
+
+## New Features
+
+[PMM-8655](https://jira.percona.com/browse/PMM-8655) - Alongside Victoria metrics, PMM now also captures QAN data during network outages. Once the connection is restored, all accumulated data is promptly delivered, ensuring you have comprehensive information to work with across your instances.
+[PMM-12459](https://jira.percona.com/browse/PMM-12459) - The [pmm_dump client utility](https://docs.percona.com/pmm-dump-documentation/index.html) previously available as a standalone client for PMM Server is now readily accessible within the PMM user interface. 
+[PMM-12463](https://jira.percona.com/browse/PMM-12463) - [Telemetry] Collecting information about env variables usage in PMM
```

Comment:
Every time you're collecting something from me, it makes me nervous. Why don't you tell me exactly what you're collecting?

Also, let's not shorten "environment".

Another tjhing, please keep things in active voice:
Percona/we now collect(s) information on environment variable usage in PMM.

Reply:
We have a doc about in-app functionality to see what's collected. 
We are not planning to keep track in RN as this is available from the pmm itself and can be changed often.